crates/fluvio-cli/src/client/hub/smartmodule/list.rs

```diff
@@ -81,7 +98,7 @@ mod output {
     impl TableOutputHandler for ListSmartModules {
         /// table header implementation
         fn header(&self) -> Row {
-            Row::from(["SMARTMODULE", "Visibility"])
+            Row::from(["SmartModule", "Visibility", "Released"])
```

Comment:
should be `published` since there there is no concept of `released` concept in hub

Reply:
Sounds good! Applied here: [ba6348a](https://github.com/infinyon/fluvio/pull/4203/commits/ba6348ad15e02909fbd3aa381f87f2df0ce26413)